user/tarsnap/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "tarsnap"
+pkgver = "1.0.40"
+pkgrel = 0
+build_style = "configure"
```

Comment:
this is gnu_configure

Reply:
are you sure? it builds fine as plain configure, but gnu_configure makes it fail with "configure.ac is required"

Reply:
that happens because the default of gnu_configure is to also run autoreconf (which gives you that error), and that requires the actual autotools inputs. since this is a very proprietary project, they removed even those (even though their actual source tree most definitely has the originals)

so what you want to also add is `configure_gen = []` to skip the reconf step- but the ./configure that is there is a gnu_configure script so you don't need to pass the --prefix=/usr. sorry if that wasn't clear :)